### PR TITLE
Add help for New/Test-FileCatalog

### LINKED ISSUE #652: MISSING documentation for New-FileCatalog, Test-FileCatalog and Install-Module -SkipPublisherCheck
I found [this explanation of the new FileCatalog cmdlets](https://msdn.microsoft.com/en-us/powershell/wmf/5.1/scenarios-features#catalog-cmdlets) in the (pre)release notes, but we are post-RTM here (in the sense that you shipped these in Windows 10 AU) and there is no help available in box, and the `-Online` URL is pointed at the old "Future Resources" page, and after some searching, the actual online help is precisely as uninformative as the in-box help.
## Can we get some documentation?

However, despite this **complete and utter lack of documentation**, the PowerShell Gallery team has apparently figured out how to **go back in time** and retroactively implement this functionality in the original 1.0.0.1 release of PowerShellGet and/or PackageManagement which have (supposedly) never been updated since release -- trying to install [Pester](/Pester/Pester) now results in a weird error indicating a new parameter `-SkipPublisherCheck` which may or may not exist:

``` posh
(Get-Command Install-Module).Parameters.ContainsKey("SkipPublisherCheck")
```

Would seem to indicate it _does_ exist, but

``` posh
C:\PS> Get-Help Install-Module -Param SkipPublisherCheck
Get-Help : No parameter matches criteria SkipPublisherCheck.
At line:1 char:1
+ Get-Help Install-Module -Param SkipPublisherCheck
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument:
(System.Manageme...CommandHelpInfo:ProviderCommandHelpInfo) [Get-Help], PSArgumentException
    + FullyQualifiedErrorId : NoParmsFound,Microsoft.PowerShell.Commands.GetHelpCommand 

```

Denies it's existence.

There is currently no documentation whatsoever (not even a synopsis with a "TODO" comment) for these core cmdlets ([New-FileCatalog](https://technet.microsoft.com/en-us/library/mt740490%28v=wps.650%29.aspx) and [Test-FileCatalog](https://technet.microsoft.com/en-us/library/mt740488%28v=wps.650%29.aspx) nor for this new parameter on Install-Module.  Nor is there any documentation within PowerShell of how these concepts relate to modules, nor of the fact that the PowerShellGet module install process checks for them...

### PR DESCRIPTION
**Pull request by joeyaiello**
Adding documentation in 5.1/6 for New/Test-FileCatalog to resolve #652

Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [x] The documented feature was introduced in version 5.1 of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.1/Microsoft.PowerShell.Security/New-FileCatalog.md](https://github.com/joeyaiello/PowerShell-Docs/blob/FileCatalog/reference/5.1/Microsoft.PowerShell.Security/New-FileCatalog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/new-filecatalog?view=powershell-5.1&branch=pr-en-us-3235) (powershell-5.1) |
[reference/5.1/Microsoft.PowerShell.Security/Test-FileCatalog.md](https://github.com/joeyaiello/PowerShell-Docs/blob/FileCatalog/reference/5.1/Microsoft.PowerShell.Security/Test-FileCatalog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/test-filecatalog?view=powershell-5.1&branch=pr-en-us-3235) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Security/New-FileCatalog.md](https://github.com/joeyaiello/PowerShell-Docs/blob/FileCatalog/reference/6/Microsoft.PowerShell.Security/New-FileCatalog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/new-filecatalog?view=powershell-6&branch=pr-en-us-3235) (powershell-6) |
[reference/6/Microsoft.PowerShell.Security/Test-FileCatalog.md](https://github.com/joeyaiello/PowerShell-Docs/blob/FileCatalog/reference/6/Microsoft.PowerShell.Security/Test-FileCatalog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/test-filecatalog?view=powershell-6&branch=pr-en-us-3235) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C2%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201811022116256643-3235%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=FBLoDeE7uMUfImTkREbHx6jf6owz%2FmGkeSFyDq9Q200%3D&st=2018-11-02T21%3A26%3A03Z&se=2018-12-03T21%3A31%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/5.1/Microsoft.PowerShell.Security/New-FileCatalog.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
-locale:  en-us
-keywords:  powershell,cmdlet
-online version:  http://go.microsoft.com/fwlink/?LinkId=822569
-external help file:  Microsoft.PowerShell.Security.dll-Help.xml
-title:  New-FileCatalog
+external help file: Microsoft.PowerShell.Security.dll-Help.xml
+keywords: powershell,cmdlet
+locale: en-us
+Module Name: Microsoft.PowerShell.Security
+ms.date: 06/09/2017
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Security/New-FileCatalog.md]: Change the date

File: reference/5.1/Microsoft.PowerShell.Security/Test-FileCatalog.md [hunk 2]
```diff
@@ -1,17 +1,24 @@
...
-locale:  en-us
-keywords:  powershell,cmdlet
-online version:  http://go.microsoft.com/fwlink/?LinkId=822570
-external help file:  Microsoft.PowerShell.Security.dll-Help.xml
-title:  Test-FileCatalog
+external help file: Microsoft.PowerShell.Security.dll-Help.xml
+keywords: powershell,cmdlet
+locale: en-us
+Module Name: Microsoft.PowerShell.Security
+ms.date: 06/09/2017
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Security/Test-FileCatalog.md]: Change the date

File: reference/6/Microsoft.PowerShell.Security/New-FileCatalog.md [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
-locale:  en-us
-keywords:  powershell,cmdlet
-online version:  http://go.microsoft.com/fwlink/?LinkId=822569
-external help file:  Microsoft.PowerShell.Security.dll-Help.xml
-title:  New-FileCatalog
+external help file: Microsoft.PowerShell.Security.dll-Help.xml
+keywords: powershell,cmdlet
+locale: en-us
+Module Name: Microsoft.PowerShell.Security
+ms.date: 06/09/2017
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Security/New-FileCatalog.md]: Change the date

File: reference/6/Microsoft.PowerShell.Security/Test-FileCatalog.md [hunk 4]
```diff
@@ -1,17 +1,24 @@
...
-locale:  en-us
-keywords:  powershell,cmdlet
-online version:  http://go.microsoft.com/fwlink/?LinkId=822570
-external help file:  Microsoft.PowerShell.Security.dll-Help.xml
-title:  Test-FileCatalog
+external help file: Microsoft.PowerShell.Security.dll-Help.xml
+keywords: powershell,cmdlet
+locale: en-us
+Module Name: Microsoft.PowerShell.Security
+ms.date: 06/09/2017
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Security/Test-FileCatalog.md]: Change the date

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.1/Microsoft.PowerShell.Security/New-FileCatalog.md](https://github.com/joeyaiello/PowerShell-Docs/blob/FileCatalog/reference/5.1/Microsoft.PowerShell.Security/New-FileCatalog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/new-filecatalog?view=powershell-5.1&branch=pr-en-us-3235) (powershell-5.1) |
[reference/5.1/Microsoft.PowerShell.Security/Test-FileCatalog.md](https://github.com/joeyaiello/PowerShell-Docs/blob/FileCatalog/reference/5.1/Microsoft.PowerShell.Security/Test-FileCatalog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/test-filecatalog?view=powershell-5.1&branch=pr-en-us-3235) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Security/New-FileCatalog.md](https://github.com/joeyaiello/PowerShell-Docs/blob/FileCatalog/reference/6/Microsoft.PowerShell.Security/New-FileCatalog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/new-filecatalog?view=powershell-6&branch=pr-en-us-3235) (powershell-6) |
[reference/6/Microsoft.PowerShell.Security/Test-FileCatalog.md](https://github.com/joeyaiello/PowerShell-Docs/blob/FileCatalog/reference/6/Microsoft.PowerShell.Security/Test-FileCatalog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/test-filecatalog?view=powershell-6&branch=pr-en-us-3235) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C2%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201811022333295050-3235%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2B4Pbm3CiTv9MS5QImvoR%2Fq%2FYnYs03GmospOOShptp84%3D&st=2018-11-02T23%3A43%3A18Z&se=2018-12-03T23%3A48%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**